src/k8s-engine/k3sHelper.ts

```diff
@@ -41,6 +41,58 @@ export interface ReleaseAPIEntry {
   }[];
 }
 
+const CURRENT_CACHE_VERSION = 2 as const;
```

Comment:
What does `as const` mean? Is that a typo? Is it even needed?

Reply:
It's TypeScript weirdness.
`const CURRENT_CACHE_VERSION = 2` means TypeScript deduces `CURRENT_CACHE_VERSION` to have a type of `number` (that is, _any_ number).
`const CURRENT_CACHE_VERSION = 2 as const` means "no really, just this one value", i.e. the type is `2` exactly, and no other number.